<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CStartWindow</class>
 <widget class="QMainWindow" name="CStartWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prime World Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="WorldNameLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="WorldGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>World</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="WorldNameSTRGLabel">
             <property name="text">
              <string>World Name STRG</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="WorldNameSTRGLineEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="DarkWorldNameSTRGLabel">
             <property name="text">
              <string>Dark World Name STRG</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="DarkWorldNameSTRGLineEdit"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="DefaultSkyboxCMDLLabel">
             <property name="text">
              <string>Default Skybox CMDL</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="DefaultSkyboxCMDLLineEdit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="WorldSAVWLabel">
             <property name="text">
              <string>World SAVW</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="WorldSAVWLineEdit"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="WorldMAPWLabel">
             <property name="text">
              <string>World MAPW</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="WorldMAPWLineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="AreaGroupBox">
           <property name="title">
            <string>Area</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="AreaSelectComboBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="AreaNameLabel">
                <property name="text">
                 <string>Area Name</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="AreaNameLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="AreaNameSTRGLabel">
                <property name="text">
                 <string>Area Name STRG</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="AreaNameSTRGLineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="AreaMREALabel">
                <property name="text">
                 <string>Area MREA</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="AreaMREALineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="AttachedAreasLabel">
                <property name="text">
                 <string>Attached Areas</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="AttachedAreasList">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="LaunchWorldEditorButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Launch World Editor</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>327</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_MLVL"/>
    <addaction name="actionExtract_PAK"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionLaunch_model_viewer"/>
    <addaction name="ActionCharacterEditor"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="ActionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_pak">
   <property name="text">
    <string>Create new...</string>
   </property>
  </action>
  <action name="actionOpen_MLVL">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExtract_PAK">
   <property name="text">
    <string>Extract PAK</string>
   </property>
  </action>
  <action name="actionLaunch_model_viewer">
   <property name="text">
    <string>Launch model editor</string>
   </property>
  </action>
  <action name="ActionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="ActionCharacterEditor">
   <property name="text">
    <string>Launch character editor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
